src/main/filters/development.properties

```diff
@@ -13,7 +13,7 @@ mongo.port=27017
 keystoneURL=http://130.206.80.57:4731/v2.0/
 adminUser=admin
 adminPass=8fa3c69e4c3e9fafa61
```

Comment:
This information should not appear here, you should leave it in blank. and define in the readme that you need to update these attributes with the proper values.